src/txTransformers.ts

```diff
@@ -93,7 +96,15 @@ const transformTxOutput = (output: TransactionOutput): TransactionOutput => {
   }
 }
 
-export const transformTxBody = (txBody: TransactionBody): TransactionBody => ({
+const transformAuxiliaryDataHash = (
```

Comment:
Maybe it should be just `computeAuxDataHash`, not transform. But it also makes sense with `transform`, just not literally.

Reply:
I'd leave it as `transform` to follow the pattern with the other fields.